src/Common/src/TypeSystem/IL/Stubs/DelegateMarshallingMethodThunk.cs

```diff
@@ -66,15 +66,60 @@ public override MethodSignature Signature
         {
             get
             {
+                if (_signature == null)
+                {
+                    PInvokeMetadata importMetadata = _invokeMethod.GetPInvokeMethodMetadata();
```

Comment:
You are right. I need to parse UnmanagedFunctionPointerAttribute, which is out of the scope of this PR. I will set isAnsi to true for now. I will fix it on the other PR which deals with delegate marshalling.